<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="UsernameInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>106</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#UsernameInput
{
	border-radius: 20px;
	border: 1px solid rgba(0, 0, 0, 100);
	padding: 10px;
	padding-left: 17px;
	font: 14pt &quot;Terminal&quot;;
}
QLineEdit#UsernameInput:hover
{

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#LoginButton
{
	border-radius: 20px;
	border: 1px solid rgba(0, 0, 0, 100);
	background-color: rgb(255, 255, 255);
	font: 14pt &quot;Terminal&quot;;
}
QPushButton#LoginButton:hover
{
	background-color: rgb(244, 244, 244);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>106</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Terminal&quot;;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>175</y>
      <width>281</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#PasswordInput
{
	border-radius: 20px;
	border: 1px solid rgba(0, 0, 0, 100);
	padding: 10px;
	padding-left: 17px;
	font: 14pt &quot;Terminal&quot;;
}
QLineEdit#PasswordInput:hover
{

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="RoleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>36</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Terminal&quot;;</string>
    </property>
    <property name="text">
     <string>Your Role:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>176</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Terminal&quot;;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="RoleComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>36</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#RoleComboBox
{
	border-radius: 20px;
	border: 1px solid rgba(0, 0, 0, 100);
	padding: 10px;
	padding-left: 17px;
	font: 14pt &quot;Terminal&quot;;
}
QComboBox#RoleComboBox:hover
{
	background-color: rgb(244, 244, 244);
}
QComboBox#RoleComboBox::drop-down:button
{
	border-radius:20px;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>User</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Creater</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ShowButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>176</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ShowButton
{
	border-radius: 20px;
	border: 0px solid rgba(0, 0, 0, 100);
	background-color: rgba(255, 255, 255, 0);
	font: 14pt &quot;Terminal&quot;;
}
QPushButton#ShowButton:hover
{
	background-color: rgb(244, 244, 244);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>view.png</normaloff>
      <activeoff>hide.png</activeoff>view.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="DropDownImg">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>DownIcon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#RegisterButton
{
	border-radius: 20px;
	border: 1px solid rgba(0, 0, 0, 100);
	background-color: rgb(255, 255, 255);
	font: 14pt &quot;Terminal&quot;;
}
QPushButton#RegisterButton:hover
{
	background-color: rgb(244, 244, 244);
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
